jib-core/src/main/java/com/google/cloud/tools/jib/frontend/CredentialRetrieverFactory.java

```diff
@@ -258,12 +258,12 @@ CredentialRetriever dockerConfig(
       try {
         Optional<Credential> credentials = dockerConfigCredentialRetriever.retrieve(logger);
         if (credentials.isPresent()) {
-          logGotCredentialsFrom("credentials from Docker config");
+          logGotCredentialsFrom("credentials from Docker config file");
           return credentials;
         }
 
       } catch (IOException ex) {
-        logger.accept(LogEvent.info("Unable to parse Docker config"));
+        logger.accept(LogEvent.info("Unable to parse Docker config file"));
```

Comment:
perhaps the actual config file, in these cases?

"credentials from Docker config file: /path/to/config/file" <-- and maybe the type too? cred helper? auths?
"Unable to parse docker config file: /path/to/config/file"

Reply:
Sounds good. Made them print the path to a Docker config file.

As for the type, I also thought that would be super useful to show here, but unfortunately it's a bit involved to have that information here. That's why I took the compromise to log that type information in `DockerConfigCredentialRetriever` as INFO. Increasing the log level will reveal the type.

Reply:
ah yeah I didn't see that.